### DISCUSSION TITLE
How to add an organization profile readme?

### DISCUSSION THREAD

**Discussion by noahstoner** [Finalgoal231/finalGoal#72] [About github] [answered]
We are owned our organization.
We add a repository named account username to add github profile readme to our github account.
But organization is different from user account.
If anyone knows a solution to add organization profile, please share it here.

**Comment by creative2113** [comment]
I understand your question. Adding a profile README to an organization on GitHub is a bit different from adding one to a personal user account.

Here's the solution for adding an organization profile README:

- 1. Create a new repository: In your organization's GitHub account, create a new repository with the name of your organization.
- 2. Add the README file: In the newly created repository, add a file named `README.md` and add the content you want to display on your organization's profile.
- 3. Make the repository public: Ensure the repository is set to "Public" so that it can be viewed by others.

That's it! Now, when users visit your organization's GitHub page, they will see the README file you've created, just like a personal user's profile README.

This is the process for adding an organization profile README on GitHub. Let me know if you have any other questions!

**Reply by creative2113** [reply]
Hello, @shinevue 
why not for me?
I am right for the very first.
but why don't you sign for me?
I am very stressful!
I can't get the anyone for a while in these days.
I want you to give me a favor.
thanks.

**Reply by noahstoner** [reply]
Please read carefully and drop your review.
Your answer is wrong.
After modify your answer, I will mark your answer.
I will only mark right answer.

**Reply by creative2113** [reply]
no problem.

**Reply by creative2113** [reply, marked as answer]
Organization profile readme is different from user's one.
1. First, create a repository named .github.
-  But user's profile readme's repository name must be equal with username.
2. Second, in folder named profile, create readme.md file
-  But user's profile readme is just readme.md file of repository that is equal with username.
3. Third, edit this readme.md. This file will be display in organization overview.
-  we can edit readme file in our mind, and if we add the organization's slogan, it is better than others.

thanks for your regarding.

**Reply by noahstoner** [reply]
Thank you for your trouble

**Reply by creative2113** [reply]
thank you very much!

**Reply by creative2113** [reply]
👍

**Comment by popstar7** [comment]
Organization profile readme is different from user's one.
First, create a repository named .github.
Second, in folder named profile, create readme.md file
Third, edit this readme.md. This file will be display in organization overview.
